doc/api/process.md

```diff
@@ -36,6 +36,18 @@ termination, such as calling [`process.exit()`][] or uncaught exceptions.
 The `'beforeExit'` should *not* be used as an alternative to the `'exit'` event
 unless the intention is to schedule additional work.
 
+```js
+function finish() {
+  console.log('It finish!!');
+}
+
+process.on('beforeExit', (code) => {
+  console.log('Process beforeExit with code:', code);
+});
+
+finish();
```

Comment:
It's not clear to me why `finish()` is put in its own function. It seems like the example code would be more clear if the work of the function was just placed right here?

Reply:
Hi @Trott, thans for your feedback. I will do my best to improve.

Reply:
@Trott of this way is better? 
```js
process.on('beforeExit', (code) => {
  console.log('Process beforeExit event with code: ', code);
});

(function doSomething () {
  console.log('This message is displayed first.');
})();
``` 
What do you think?

Reply:
Why does the `console.log()` message inside of `doSomething()` need to be inside of `doSomething()`? Can't it just be `console.log('This message is displayed first.');` without any function wrapping? Or does the function-wrapping serve some purpose?